### PR TITLE
Update belong_in_tech.md

### PR DESCRIPTION
**Pull request by Jstewart3313**
@emhickmann

### CODE REVIEW

**Comment by Jstewart3313** [comment]
Please let us know about the round robin idea.

File: m4_sessions/1811_inning/belong_in_tech.md [hunk 1]
```diff
@@ -1,30 +1,75 @@
...
 
 
 
 ## Name of Session: 
+Why you belong tech.
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Have you ever felt like an outsider? Is your background not tech related? Come find out why the tech industry really needs you.
```
Comment (emhickmann) [inline m4_sessions/1811_inning/belong_in_tech.md:14]: Great description

File: m4_sessions/1811_inning/belong_in_tech.md [hunk 2]
```diff
@@ -1,30 +1,75 @@
...
-     Share resources in your slack channel
-     All sessions NEED to have a closing activity, reflection, discussion
-     # Introductons 
-     # Intro to new material
-     # Small Group Activity
-     # Closing Discussion
+### Recommended Trajectory 
+     
+ Introduction - and split class into 4 small groups
+ Acticity-
```
Comment (emhickmann) [inline m4_sessions/1811_inning/belong_in_tech.md:31]: This is an awesome opening.  Please add your specific content (either talking points, questions, or even content that you want to cover).  Think about leaving a session that other people could leverage.

File: m4_sessions/1811_inning/belong_in_tech.md [hunk 3]
```diff
@@ -1,30 +1,75 @@
...
+Andrew - Positive intelligence ( reference the Book) Silence negative voices.
+
+** each 10 minutes for each round robin^^
+
+fill time for character description
+     did you perspective change with the new descriptions shared by peers
+
+
+closing - 
+5 minutes on reflection
```
Comment (emhickmann) [inline m4_sessions/1811_inning/belong_in_tech.md:68]: You can combine the reflection and survey time as they will just do a reflection, no survey

File: m4_sessions/1811_inning/belong_in_tech.md [hunk 4]
```diff
@@ -1,30 +1,75 @@
...
+Justin - Your difference does not matter
+Andrew - Positive intelligence ( reference the Book) Silence negative voices.
+
+** each 10 minutes for each round robin^^
+
+fill time for character description
+     did you perspective change with the new descriptions shared by peers
+
+
+closing -
```
Comment (emhickmann) [inline m4_sessions/1811_inning/belong_in_tech.md:67]: What are your closing comments?  What do you want people to take away?  Are people sharing whole group?

File: m4_sessions/1811_inning/belong_in_tech.md [hunk 5]
```diff
@@ -1,30 +1,75 @@
...
+count off by 4's and break out into small groups lead by each moderator. 2 minutes
+
+Jessica - Diversity statistics and why they matter
+Mark - code switching, and why its ok. 
+Justin - Your difference does not matter
+Andrew - Positive intelligence ( reference the Book) Silence negative voices.
+
+** each 10 minutes for each round robin^^
+
+fill time for character description
```
Comment (emhickmann) [inline m4_sessions/1811_inning/belong_in_tech.md:63]: This is great.  Are they talking to each other in pairs?  Or in small groups?

**Review by emhickmann — COMMENTED** [review]
@Jstewart3313 I think this is a great start.  Use the next section to really refine the break out groups and know what content each of you are covering.

File: m4_sessions/1811_inning/belong_in_tech.md (hunk 4 above)
Comment (Jstewart3313) [reply m4_sessions/1811_inning/belong_in_tech.md:67]: reflection questions
     * take a minute to think about your life track towards getting you into tech
     * How can your different experiences and skills benefit a team.
     * What did you learn today that you can incorporate into your daily thought process and into the next inning
     * is there something that we could add to this gear up to help get of over mental roadblocks
     * As developer how will you try to make everyone feel welcome and accepted

File: m4_sessions/1811_inning/belong_in_tech.md (hunk 3 above)
Comment (Jstewart3313) [reply m4_sessions/1811_inning/belong_in_tech.md:68]: Added multiple questions for journal entries to reflect solo on how the gear up can change them as a developer in the workplace in the future.

File: m4_sessions/1811_inning/belong_in_tech.md (hunk 2 above)
Comment (Jstewart3313) [reply m4_sessions/1811_inning/belong_in_tech.md:31]: Will open the floor to get some more understanding to the class. ask why they chose to attend our group, or if they have any fears or doubts themselves.

**Pull request merged**